### DISCUSSION TITLE
顧問弁護士さんからのコメント (2023/11/6)

### DISCUSSION THREAD

**Discussion by sogaoh** [sre-next/corporate-regulations#13] [Ideas]
規程管理規程

■４条１
「前条に定める諸規程は、前条に定める順序により上位規程とし、」と定める点、３条の01,02,…の番号の少ないものが上位という整理でしょうか？
つまり、01>02>02…ということです。ただ、一般的には01～07に記載の規程はいずれも並列の関係になるかと思いますので、理解が異なるようでしたらお知らせください。
異なる場合、少々４条１項に追記が必要と思われます。

■１６条
「規程管理者のうち 1 名以上のレビュー」をもって改廃との点について、当該レビュー担当者は拒否権を持つでしょうか？
つまり、レビュー担当者が改廃すべきでないとした場合には改廃ができないというルールでしょうか、それとも意見は参考に過ぎず、改廃の最終決定は、責任者の一存で可能としますでしょうか？いずれも法的には可能ですが改廃権限に関わる重要な点なので、規程上明記しておく必要があるかと思っております。

**Comment by sogaoh** [comment]
@okash1n @katsuhisa91 お手すきでコメントいただければと

> ■４条１

これ、「前条」じゃなくて「前項」ですかね？ 


> ■１６条

規程管理者の合議、かなと思いますが、どう表現するのが妥当なんでしょう。。

**Reply by katsuhisa91** [reply]
■４条１

弁護士さんのコメントは、「前条」に対するコメントとの理解でよいのではないでしょうか。
三条に記載されている順番、すなわち「条」の番号が若いものを上位規程と位置づけているが、一般的には並列の解釈をすべきではとコメントをいただいたと理解しています。
たしかに私もそうな気がしてきました。
一条の基本規程が上位概念にすわるのはともかく、それ以外は並列の位置づけであるという理解で書き換えるのが妥当かもしれません。

■１６条

規程管理者の過半数の合意をもって・・・という表現が妥当な気がしますね。
